wbooks/src/app/screens/Library/index.tsx

```diff
@@ -16,6 +19,13 @@ interface Props {
 }
 
 function Library({ navigation }: Props) {
+  const dispatch = useDispatch();
+  const { books, error } = useSelector<RootState, BookState>(state => state.books);
+
+  React.useEffect(() => {
+    dispatch(actionCreators.getBooks());
+  }, [dispatch, books, error]);
```

Comment:
Why `books` and `error` are dependencies?

Reply:
Both removed